app/controllers/users_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
   # GET /users/1
   # GET /users/1.json
   def show
```

Comment:
- This is where we need to show past and upcoming events for both created events and attended events through our created `attended_events` association for attendations and `events` for created events So they will be available in the views.

Reply:
```
4. Make a User’s Show page display a list of events they are attending.
```
- this requirement is quite explicit - the user's show page has nothing to do with events they created, but rather events they are associated with in form of attendance.
- also for distinguishing between events, I had earlier on added a calendar icon - to distinguish between attended events (with green icon) from upcoming events (with yellow icon)

Reply:
Also quoting from Odin specification:

![Screenshot from 2019-10-23 13-06-59](https://user-images.githubusercontent.com/4246637/67386842-01773580-f596-11e9-895e-aac477e6fa3e.png)

You can find it under "Basic Events" section

Reply:
Thanks, I will go ahead and implement it.

Reply:
Please check this error: #2.